docs/js-client-sdk.getinstance.md

```diff
@@ -4,6 +4,10 @@
 
 ## getInstance() function
 
+> Warning: This API is now obsolete.

```

Comment:
same question

Reply:
also whenever deprecating something, we should always propose an alternative so users are not left guessing.

Reply:
This file is auto-generated, so the text is out of my hands. I have, however, updated all of the deprecated tags with an alternative. thank you